tools/travis-test.sh

```diff
@@ -215,7 +217,7 @@ build_pkg () {
   local platform=$1
   cd tools/pkg
   ./build $platform
-  ./run $platform
+  ./run $platform $GIT_REF
```

Comment:
From what I can see the script accepts 3rd parameter named `REVISION`. Can we calculate the revision here (as `GIT_REF`) and pass it to the script? It would make the revision consistent for all platforms.